host/xtest/ffa_spmc_1000.c

```diff
@@ -62,6 +67,47 @@ static struct ffa_ioctl_ep_desc test_endpoint3 = {
 	.uuid_ptr = (uint64_t)test_endpoint3_uuid,
 };
 
+static bool check_ffa_user_version(void)
+{
+	FILE *f = NULL;
+	int ver_major = -1, ver_minor = -1, ver_patch = -1, scan_cnt = 0;
```

Comment:
Please declare each variable separately on one line each.

Reply:
Thank you for the finding! I have just corrected it.